mkosi.images/system/mkosi.conf.d/10-fedora/mkosi.conf

```diff
@@ -10,8 +10,6 @@ Packages=
         dnf5
         f2fs-tools
         scsi-target-utils
-        kernel-modules-extra
-        kernel-modules-internal
```

Comment:
Please keep them for fedora, or at least one distribution. Otherwise, many test cases for networkd are skipped.

Reply:
@yuwata Can you get me a list of missing kernel modules in kernel-modules-core so I can go yell at the Fedora kernel maintainer to add them to kernel-modules-core?

Reply:
I do not think modules used by networkd tests should be moved to kernel-modules-core.
E.g. netdevsim module, which is in -internal package, is not necessary in general, as it is a kind of emulator.

Reply:
Another example is sch_netem. It is neither necessary in general for production.

Reply:
@yuwata But as long as these modules are not too big and don't do anything unless loaded, there shouldn't be any harm in having them in kernel-modules-core right?

Reply:
Not only the two modules, but many sch_xyz required by the test. Even if each module is small enough, including many small mostly uncommon modules is not good, I think.